src/components/TeamsPricingCard.astro

```diff
@@ -0,0 +1,76 @@
+---
+import Tooltip from "./Tooltip.astro";
+
+export interface Props {
+  title: string;
+  description: string;
+  price: string;
+  price_details: string;
+  payment_details: string;
+  features: string[];
+  feature_tooltip: string[];
+}
```

Comment:
Nitpick: let's use camelCase for the attributes, since that's the usual JS style.